base/deprecated.jl

```diff
@@ -1298,6 +1298,10 @@ end
      end
 end
 
+# #19635
+@deprecate zeros{T <: Number}(::Type{T}, arr::Range)  zeros(T, length(arr))
```

Comment:
Right. In fact a glitch with `zeros(T, arr)` #19265 was the trigger for the zeros PR #19635.